.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ concurrency:
 jobs:
   puppet:
     name: Puppet
-    uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v1
+    uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v2
     with:
       pidfile_workaround: 'false'
+      beaker_facter: 'nodejs_version:NodeJS:16,18,20'
```

Comment:
Technically, NodeJS 16 is EOL but it's the only version that is supported on EL7 by nodesource :woman_shrugging: